content/v2/domains/dnssec.markdown

```diff
@@ -0,0 +1,228 @@
+---
+title: DNSSEC API | Domains | DNSimple API v2
+excerpt: This page documents the DNSimple DNSSEC API v2.
+---
+
+# DNSSEC API
+
+* TOC
+{:toc}
+
+
```

Comment:
I've added the beta banner and a note in the create delegation signer endpoint regarding when it should be used.